### PR TITLE
Add simplified ecobenefits

### PR DESCRIPTION
**Pull request by sireliah**

### CODE REVIEW

File: opentreemap/treemap/views/tree.py [hunk 1]
```diff
@@ -118,6 +120,49 @@ def search_tree_benefits(request, instance):
     return context
 
 
+def simplified_tree_benefits(request, instance):
+    unit = 'kg/rok'
```
Comment (vevurka) [inline opentreemap/treemap/views/tree.py:124]: Should it be hardcoded? What about possible translations in the future?

File: opentreemap/treemap/views/tree.py [hunk 2]
```diff
@@ -118,6 +120,49 @@ def search_tree_benefits(request, instance):
...
+    req = requests.get(settings.ECO_SERVICE_URL)
+    req.raise_for_status()
+    data = req.json()
+
+    descriptions = {
+        'O3': u'Redukcja ozonu (O3)',
```
Comment (vevurka) [inline opentreemap/treemap/views/tree.py:130]: Same here regarding hardcoding polish phrases.

File: opentreemap/treemap/views/tree.py [hunk 3]
```diff
@@ -118,6 +120,49 @@ def search_tree_benefits(request, instance):
...
+        'NO2': u'Redukcja dwutlenku azotu (NO2)',
+        'SO2': u'Redukcja dwutlenku siarki (SO2)',
+        'PM2.5': u'Redukcja pyłów 2.5 µm (PM2,5)'
+    }
+
+    benefits = {
+        k.replace('.', ''):
+        {
+            'label': descriptions[k],
+            'value': round(v / 1000, 2),
```
Comment (vevurka) [inline opentreemap/treemap/views/tree.py]: Why we divide by 1000?

File: opentreemap/treemap/views/tree.py [hunk 4]
```diff
@@ -118,6 +120,49 @@ def search_tree_benefits(request, instance):
...
+            'unit': unit,
+        }
+        for k, v in data['benefits'].iteritems()
+    }
+    benefits['totals'] = {
+        'label': u'Redukcja zanieczyszczeń w sumie',
```
Comment (vevurka) [inline opentreemap/treemap/views/tree.py:146]: Translations?

File: opentreemap/treemap/views/tree.py [hunk 5]
```diff
@@ -118,6 +120,49 @@ def search_tree_benefits(request, instance):
...
+        'value': reduce(lambda x, row: x + row['value'], benefits.values(), 0),
+        'unit': unit
+    }
+
+    context = {
+        'used_trees': data['trees_count'],
+        'total_trees': Tree.objects.filter(instance=instance).count(),
+        'n_empty_plots': 0,
```
Comment (vevurka) [inline opentreemap/treemap/views/tree.py]: What is `n_empty_plots` and why do we need to set it to 0? Why do we need `n_resources` set to 0? Could that be that setting `has_resources` to False would make these fields not longer required?

**Review by vevurka — COMMENTED** [review]
Some small details which I would like to be clarified/addressed before approval. Overall looks good.

File: opentreemap/treemap/views/tree.py (hunk 3 above)
Comment (sireliah) [reply opentreemap/treemap/views/tree.py]: The `ecobenefits` returns values in gram units, however grams don't look really meaningful in the frontend. I'll put a comment about this.

File: opentreemap/treemap/views/tree.py (hunk 1 above)
Comment (sireliah) [reply opentreemap/treemap/views/tree.py:124]: In fact this fork is merely a proof of concept for the Polish map, so there is no use in maintaining translations. It's also extremely complicated to manage translations in this project, I hope you remember. :D

File: opentreemap/treemap/views/tree.py (hunk 2 above)
Comment (sireliah) [reply opentreemap/treemap/views/tree.py:130]: Yeah, we'll not maintain translations here.

File: opentreemap/treemap/views/tree.py (hunk 4 above)
Comment (sireliah) [reply opentreemap/treemap/views/tree.py:146]: Same as above.

File: opentreemap/treemap/views/tree.py (hunk 5 above)
Comment (sireliah) [reply opentreemap/treemap/views/tree.py]: I decided to keep those entries in the context dict so there is no need to modify html templates. However I can try to remove most of redundant entries if this doesn't break the view.

**Review by vevurka — APPROVED** [review]

**Pull request merged**